apps/admin_api/lib/admin_api/v1/controllers/minted_token_controller.ex

```diff
@@ -53,7 +53,7 @@ defmodule AdminAPI.V1.MintedTokenController do
   @doc """
   Creates a new Minted Token.
   """
-  def create(%{assigns: %{account: account}} = conn, attrs) do
+  def create(%{assigns: %{auth_account: account}} = conn, attrs) do
```

Comment:
Aren't we removing this entirely in PR #178?

Reply:
Yes but they're separate PR so I can't remove this patten matching here.